docs/README.md

```diff
@@ -0,0 +1,38 @@
+# Service Manager CLI Documentation
+
+The Service Manager CLI is the official tool to communicate with a Service Manager instance directly.
+
+> **NOTE**: Throughout this document we will refer to the Service Manager CLI as *SM CLI* for simplicity.
+
+## Getting Started
+In order to start using the SM CLI you need to download and install it. To do that follow the instructions [HERE][1].
+
+## Commands
+The SM CLI provides commands for creating, listing, updating and deleting service brokers and platforms in a Service Manager instance. Here's a full list of the available commands:
+
+* [login][2]
+* [register-broker][3]
+* [update-broker][4]
+* [list-brokers][5]
+* [delete-broker][6]
+* [register-platform][7]
+* [update-platform][8]
+* [list-platforms][9]
+* [delete-platform][10]
+* [info][11]
+* [version][12]
+* [help][13]
+
+[1]: https://github.com/Peripli/service-manager-cli/blob/master/README.md 
```

Comment:
Use relative links instead, because these do not work at the moment
[command](commands/login.md)